src/pybind/mgr/progress/module.py

```diff
@@ -738,6 +745,42 @@ def shutdown(self):
         self._shutdown.set()
         self.clear_all_progress_events()
 
+    def _process_avg_clone_progress(self):
+        '''
+        Collect all on-going events (which are stored in self._events)
```

Comment:
I guess this part handles (remote) events scheduled by the volumes plugin - the part we discussed in slack related to aggregating the stats in the volumes module itself and concluded to aggregate in the volumes module itself.

Reply:
Done.